src/shared/containers/Dashboard/index.jsx

```diff
@@ -45,62 +49,68 @@ function SlashTCContainer(props) {
                 tags={[INNOVATION_CHALLENGES_TAG]}
                 challengeListingQuery={challengeListingQuery}
                 tracks={[]}
                 itemCount={20}
               />
-              <ChallengesFeed theme="dark" excludeTags={[INNOVATION_CHALLENGES_TAG]} />
               {/* deprected with https://topcoder.atlassian.net/browse/CORE-346 */}
               {/* <GigsFeed itemCount={5} theme="dark" /> */}
-              <NewsFeed />
-              <Viewport id="SSwOFPT8l0WpGhqCBRISG" />
-              <ThriveArticlesFeedContainer itemCount={4} theme="dark" />
-              <BlogFeedContainer itemCount={4} theme="dark" />
-              <Viewport id="6sjlJHboX3aG3mFS5FnZND" />
+              {/* deprecated with https://topcoder.atlassian.net/browse/TOP-1390 */}
+              {/* <NewsFeed /> */}
+              { isDevEnv ? <Viewport id="2qVJTorSdRVNlfRqoQocUH" /> : <Viewport id="SSwOFPT8l0WpGhqCBRISG" /> }
+              <ThriveArticlesFeedContainer itemCount={4} theme="light" />
+              <BlogFeedContainer itemCount={4} theme="light" />
+              { isDevEnv ? <Viewport id="2tq6jtu9GzPab7lAb7swlT" /> : <Viewport id="6sjlJHboX3aG3mFS5FnZND" /> }
             </div>
           </div>
         ) : (
           <div className={theme.layoutWrapper}>
             {/* Left column */}
             <div className={theme.column}>
               <TopcoderTime />
-              <ThriveArticlesFeedContainer itemCount={4} theme="dark" />
-              <BlogFeedContainer itemCount={4} theme="dark" />
-              <Viewport id="6sjlJHboX3aG3mFS5FnZND" />
+              <ThriveArticlesFeedContainer itemCount={4} theme="light" />
+              <BlogFeedContainer itemCount={4} theme="light" />
+              { isDevEnv ? <Viewport id="2tq6jtu9GzPab7lAb7swlT" /> : <Viewport id="6sjlJHboX3aG3mFS5FnZND" /> }
             </div>
             {/* Center column */}
             <div className={theme.column}>
-              <Viewport id="1BK50OyMT29IOavUC7wSEB" />
+              { isDevEnv ? <Viewport id="IYMEHgYwk6S0S9tx5SsHd" /> : <Viewport id="1BK50OyMT29IOavUC7wSEB" /> }
               <ChallengesFeed
-                theme="dark"
-                title="INNOVATION CHALLENGES"
+                theme="light"
+                excludeTags={[INNOVATION_CHALLENGES_TAG]}
+              />
+              <ChallengesFeed
+                theme="light"
+                title="Innovation Challenges"
                 tags={[INNOVATION_CHALLENGES_TAG]}
                 challengeListingQuery={challengeListingQuery}
                 tracks={[]}
                 itemCount={20}
               />
-              <ChallengesFeed theme="dark" excludeTags={[INNOVATION_CHALLENGES_TAG]} />
               {/* deprected with https://topcoder.atlassian.net/browse/CORE-346 */}
               {/* <GigsFeed itemCount={5} theme="dark" /> */}
-              <NewsFeed />
+              {/* deprecated with https://topcoder.atlassian.net/browse/TOP-1390 */}
```

Comment:
Same question -- why comment this out instead of just removing it?

Reply:
@kkartunov :point_up: